docs/developer-docs/latest/developer-resources/database-apis-reference/rest/relations.md

```diff
@@ -0,0 +1,238 @@
+  }
+}
+
+// Using the longhand syntax
+{
+  data: {
+    categories: { set: [{ id: 2 }, { id: 4 }] }
+  }
+}
+```
+
+Using the longhand syntax with `connect` allows adding [positional arguments](#longhand-syntax) to define an order for relations.
+
+## `connect`
+
+Using `connect` in the body of a request performs a partial update, adding specified relations.
+
+`connect` accepts either a shorthand or a longhand syntax.
+
+### Shorthand syntax
+
+With the shorthand syntax, pass an array of ids of relations to create.
+
+**Example:**
+
+Passing the following example of body request adds the `categories` entries with ids `2` and `4` to a content-type's relations:
+
+```json
+{
+  data: {
+    categories: { connect: [2, 4] }
+  }
+}
+```
+
+### Longhand syntax
+
+With the longhand syntax, you can pass optional positional arguments to define the order of relations.
+
+The longhand syntax accepts an array of objects, each object containing the `id` of the relation to be connected and an optional `position` object to define where to connect the relation.
+
+`position` accepts 4 different positional attributes:
+
+| Parameter name and syntax | Description |
+|---------------------------|-------------|
+| `before: id`              | Positions the relation before the given `id`. |
+| `after: id`               | Positions the relation after the given `id`. |
+| `start`                   | Positions the relation at the start of the existing list of relations. |
+| `end`                     | Positions the relation at the end of the existing list of relations. |
+
+The `position` argument is optional and defaults to `position: { end: true }`.
+
+:::note Sequential order
+Since `connect` is an array, the order of operations is important as they will be treated sequentially (see combined example below).
+:::
+
+**Examples:**
+
+:::: tabs card
+
+::: tab Basic example
+
+Given the following record in the database:
+
+```json
+categories: [
+  { id: 1, order: 1 }
+  { id: 2, order: 2 }
+]
+```
+
+sending the following in the request body of a PUT or POST request will create a third relation of `id` `3` and position it before the relation with `id` `2`:
+
+```json
+categories: {
+  connect: [
+    { id: 3, position: { before: 2 } }, // It should be placed before relation id=2
+  ]
+}
+```
+
+:::
+
+::: tab Combined example
+
+Sending the following example in the request body of a PUT or POST request updates multiple relations:
+
+```json
+categories: {
+  connect: [
+    // By default positional info will be end: true
+    { id: 6, position: { after: 1} },    // It should be after relation with id 1
+    { id: 7, position: { before: 2 }},   // It should be before relation with id 2
+    { id: 8, position: { end: true }},   // It should be at the end
+    { id: 9 },                           // It should also be at the end
+    { id: 10, position: { start: true }} // It should be at the start
+  ]
+}
+```
+
+The resulting database record could be the following:
```

Comment:
Thanks! I wasn't sure because I think the database record could include more entries, actually 😅  To be checked with @Marc-Roig to ensure I didn't misinterpret the result of the various connect operations :)

Reply:
Agree, "will" would be more appropriate here.

>Thanks! I wasn't sure because I think the database record could include more entries
That is also true, the result depends on the initial relations on the database. Looking at it, this example won't work when creating a new entity:

```js
categories: {
  connect: [
    // By default positional info will be end: true
    { id: 6, position: { after: 1} },    // < ==== ID 1 does not exist in this list. It will only work if it is in the database already.
    { id: 7, position: { before: 2 }},   // Same for id 2
    { id: 8, position: { end: true }},  
    { id: 9 },                           
    { id: 10, position: { start: true }} 
  ]
}
 ```

 
This example would be only valid if you are editing a relationship that previously was:
 ```js
categories: [
  { id: 1, order: 1},
  { id: 2, order: 2},
]
 ```

Would it be useful to mention this difference between the PUT / POST here? Or creating an example for the POST and another for the PUT?

Reply:
Thanks, @Marc-Roig. Is there a big difference between PUT and POST regarding to relations (except the nature of the operation itself, of course)?
Because this documentation is not a tutorial, we don't need to be fully exhaustive, but if there's an important difference (syntax), or something that would work with one method and not the other, we should highlight it, indeed. In this case, could you please give me a precise example below 👇 (original database record, if relevant, + excerpt of the request body to pass, + final result)? Thank you!

Reply:
Sure, thanks, that's why I updated the text a bit. :)